src/ui.js

```diff
@@ -0,0 +1,79 @@
+const inputField = document.querySelector('.input-field');
+const list = document.querySelector('#list-wrap');
+const counter = document.querySelector('#count');
+
+let todoArr = [];
+const todo = {
+  description: String,
+  completed: false,
+};
+
+export default class UI {
+  static addTodo() {
+    if (inputField.value.length !== 0) {
+      todo.description = inputField.value;
+      todoArr.push(todo);
+      inputField.value = '';
+      for (let i = 0; i < todoArr.length; i += 1) {
+        todoArr[i].index = i + 1;
+      }
+      localStorage.setItem('todos', JSON.stringify(todoArr));
+    }
+  }
+
+  static displayTodos() {
+    const todos = JSON.parse(localStorage.getItem('todos'));
+    counter.textContent = todos.length;
+    if (todos) {
+      todoArr = todos;
+    }
+    let result = '';
+    todoArr.map((todo) => {
+      result += `
+        <li id=${todo.index} class="list-item">
+            <form id=${todo.index} class='edit-todo' action="">
+                <div class="label-wrap">
+                    <label for="checkbox">
+                        <input class="checkbox" type="checkbox" name="checkbox" id=${todo.index}>
+                    </label>
+                    <label for=${todo.index}>
+                        <input class="input" type="text" name="add-item" id=${todo.index} value="${todo.description}">
+                    </label>
+                </div>
+                </form>
+                <button id=${todo.index} class="drag">remove</button>
+        </li>
+              `;
+      return result;
+    });
+
+    list.innerHTML = result;
+    return result;
+  }
+
+  static removeTodo(e) {
+    const deleteBtns = [...document.querySelectorAll('.drag')];
+    deleteBtns.forEach((btn) => {
+      if (e.target === btn) {
+        const filtered = todoArr.filter((item) => item.index.toString() !== e.target.id.toString());
+        todoArr = filtered;
+        localStorage.setItem('todos', JSON.stringify(todoArr));
+        UI.displayTodos();
+      }
+    });
+  }
```

Comment:
- [ ] Great job implementing all the core functionalities. :+1: Let's work on some minor detail to achieve the objective of this project. Kindly update indexes of the remaining items when a to-do is removed. ✍️

**Project Requirements**
![Screenshot from 2023-02-24 11-40-09](https://user-images.githubusercontent.com/17243325/221133635-d07f301a-0069-40c0-81cf-45a8ecbb2786.png)

Reply:
> * [ ]  Great job implementing all the core functionalities. 👍 Let's work on some minor detail to achieve the objective of this project. Kindly update indexes of the remaining items when a to-do is removed. ✍️
> 
> **Project Requirements** ![Screenshot from 2023-02-24 11-40-09](https://user-images.githubusercontent.com/17243325/221133635-d07f301a-0069-40c0-81cf-45a8ecbb2786.png)

Hi @amejid,
 Thanks for your feedback. I have  now implemented the above requirement.